tests/no_create_accessanalyzer/main.tf

```diff
@@ -0,0 +1,14 @@
+provider "aws" {
+  region = "us-east-1"
+}
+
+module "create_iam_account" {
+  source = "../../"
+
+  providers = {
+    aws = aws
+  }
+
+  create_iam_account    = true
+  create_accessanalyzer = false
+}
```

Comment:
Given that this doesn't have a new line at the end of the file, linting should not be passing. Please add a line to the end of this file and then run `make docker/run target=eclint/lint`

Reply:
just pushed a new version of tardigrade-ci (0.0.7) that should address this